### PR TITLE
bug fix wrong folder kickstart

### LINKED ISSUE #394: wrong folder kickstart
**What happened**:
```
$ ./local-up-karmada.sh 
Creating cluster "karmada-host" ...
 ✓ Ensuring node image (kindest/node:v1.21.1) 🖼 
 ✓ Preparing nodes 📦  
 ✓ Writing configuration 📜 
 ✓ Starting control-plane 🕹️ 
 ✓ Installing CNI 🔌 
 ✓ Installing StorageClass 💾 
 ✓ Waiting ≤ 2m0s for control-plane = Ready ⏳ 
 • Ready after 30s 💚
Set kubectl context to "kind-karmada-host"
You can now use your cluster with:

kubectl cluster-info --context kind-karmada-host --kubeconfig /root/.kube/karmada.config

Have a nice day! 👋
Context "kind-karmada-host" renamed to "karmada-host".
Cluster "kind-karmada-host" set.
make: *** No rule to make target 'images'.  Stop.
$ docker ps
CONTAINER ID   IMAGE                                        COMMAND                  CREATED              STATUS              PORTS                        NAMES
f74e81970c4e   kindest/node:v1.21.1                         "/usr/local/bin/entr…"   About a minute ago   Up About a minute   127.0.0.1:46743->6443/tcp    karmada-host-control-plane
```
**What you expected to happen**:
either choose to continue on a different context or stop earlier!

**How to reproduce it (as minimally and precisely as possible)**:
```
$ git clone ; cd karmada/hack ; ./local-up-karmada.sh 
```

### PR DESCRIPTION
**Pull request by lfbear**
Signed-off-by: lfbear <lfbear@gmail.com>

**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:

fix bug when startups `local-up-karmada.sh` in the path `karmada/hack/`

**Which issue(s) this PR fixes**:
Fixes #394

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

NONE

### CODE REVIEW

File: hack/local-up-karmada.sh [hunk 1]
```diff
@@ -28,7 +32,7 @@ fi
 # make controller-manager image
 export VERSION="latest"
 export REGISTRY="swr.ap-southeast-1.myhuaweicloud.com/karmada"
-make images
+make images --directory="${SCRIPT_ROOT}"
```
Comment (RainbowMango) [inline hack/local-up-karmada.sh:35]: Wow, I didn't know the option before. Thanks~

**Comment by RainbowMango** [comment]
/lgtm
/approve

/hold
for a while, see if @theonlydoo has any comments.

**Comment by RainbowMango** [comment]
> /lgtm

Take this as agreement from @theonlydoo .
/hold cancel

@theonlydoo really glad to have you here. I guess the membership guideline will coming soon, you can request to become a member of the community then.

File: hack/local-up-karmada.sh (hunk 1 above)
Comment (lfbear) [reply hack/local-up-karmada.sh:35]: This option just for our requirement

```
make --help
Usage: make [options] [target] ...
Options:
  -C DIRECTORY, --directory=DIRECTORY
                              Change to DIRECTORY before doing anything.
```

Comment (RainbowMango) [reply hack/local-up-karmada.sh:35]: Got it!

**Pull request merged**